docs/spec/openapi.yml

```diff
@@ -1858,7 +1959,44 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-
+  '/data-importers':
+    x-everest-resource-name: data-importers
+    get:
+      tags:
+        - Data Importer
+      summary: List data importers
+      description: |
+        This API lists all data importers in the kubernetes cluster.
+      operationId: listDataImporters
+      parameters:
+      - in: query
+        name: supportedEngine
+        description: Filter data importers by supported database engine type. 
```

Comment:
comma-separated?

Reply:
yeah, the `style: form` does that. Does it matter? Not sure what the question / concern was

Reply:
if we accept comma-separated values - it shall be stated in API docs. It helps to understand API for user

Reply:
Done